### ISSUE TITLE
DST Staging Review (Consider): Alt text guidance seems out of place

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#3676]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** Alert Sign-in
**Staging Review ticket:** [Design System Staging Review: Alert Sign-in](https://github.com/department-of-veterans-affairs/va.gov-team/issues/99891)

## Findings details
The details of this ticket are for consideration only. No Staging Review issue was found.
**High-priority:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

As currently written, the accessibility guidance includes information on when alt text is appropriate for any images in the alerts.

In the screenshot examples, the only images are the branding iconography for ID.me and Login.gov. These don't seem like situations where alt text is appropriate.

## Recommended action
Is there ever a situation where one of these alerts should have an image?

If not, I wonder if it would be clearer to just state in the guidance that the only images that can be used in the component are branding iconography provided by the authentication providers, and give an example for how a button with a Login.gov or ID.me icon should be coded. All of the decorative vs informative information is correct, but if we don't expect that to ever be an issue I feel like including that guidance could lead teams to second-guess themselves or to think something's permissable that's not.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for DST

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation
